crates/relayer/src/link/relay_path.rs

```diff
@@ -108,12 +108,17 @@ pub struct RelayPath<ChainA: ChainHandle, ChainB: ChainHandle> {
     // transactions if [`confirm_txes`] is true.
     pending_txs_src: PendingTxs<ChainA>,
     pending_txs_dst: PendingTxs<ChainB>,
+
+    pub max_memo_size: usize,
+    pub max_receiver_size: usize,
 }
 
 impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
     pub fn new(
         channel: Channel<ChainA, ChainB>,
         with_tx_confirmation: bool,
+        max_memo_size: usize,
```

Comment:
Same here

Reply:
Fixed in f2b3a4fc46891f7f047dfb3a6336bf2a31409b59